### PR TITLE
Fix egg.json

### PR DESCRIPTION
**Pull request by ebebbington**
**Description**

* Fix egg.json (again)

### CODE REVIEW

File: egg.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "./create_app.ts",
     "./src/**/*",
     "./README.md",
+    "./mod.ts",
```
Comment (crookse) [inline egg.json:13]: mod.ts twice?

Comment (ebebbington) [reply egg.json:13]: oh yeah.. huh :L good spot ill remove it

Comment (ebebbington) [reply egg.json]: https://github.com/drashland/deno-drash/pull/388/commits/c2b0623cd0d1a38ce62e87bbe5f69f0c77452df2

**Review by crookse — COMMENTED** [review]
0 files changed? lol

**Comment by ebebbington** [comment]
wtf xD

**Comment by ebebbington** [comment]
should i just open a new pr or something? this makes no sense :S

**Comment by crookse** [comment]
@ebebbington, lmao yea

**Pull request closed without merging**